### ISSUE TITLE
center_loss_factor and prelogits_norm_loss_factor during softmax training

### ISSUE THREAD

**Issue by w19787** [davidsandberg/facenet#1161]
In train_softmax.py, the default value of center_loss_factor and prelogits_norm_loss_factor are both 0.0. Is it the setting we should use to train facenet from scratch?

![image](https://user-images.githubusercontent.com/731496/85501454-fa6a6080-b617-11ea-952c-3585720303f9.png)
